varken/overseerr.py

```diff
@@ -0,0 +1,177 @@
+        movie_endpoint = '/api/v1/movie/'
+        tv_endpoint = '/api/v1/tv/'
+
+        #GET THE LATEST n REQUESTS
+        req = self.session.prepare_request(Request('GET', self.server.url + endpoint))
+        get_latest_req = connection_handler(self.session, req, self.server.verify_ssl)
+
+        #RETURN NOTHING IF NO RESULTS
+        if not get_latest_req:
+            return
+
+        influx_payload = []
+
+        # Request Type: Movie = 1, TV Show = 0
+        for result in get_latest_req['results']:
+            if result['type'] == 'tv':
+                req = self.session.prepare_request(Request('GET', self.server.url + tv_endpoint + str(result['media']['tmdbId'])))
+                get_tv_req = connection_handler(self.session, req, self.server.verify_ssl)
+                hash_id = hashit(f'{get_tv_req["id"]}{get_tv_req["name"]}')
+
+<<<<<<< HEAD
+=======
+                print(result)
+
+>>>>>>> 07cc5946f37b14ab268c795fbdc5092988c10e97
+                influx_payload.append(
+                    {
+                        "measurement": "Overseerr",
+                        "tags": {
+                            "type": "Requests",
+                            "server": self.server.id,
+                            "request_type": 0,
+                            "status": get_tv_req['mediaInfo']['status'],
+                            "title": get_tv_req['name'],
+                            "requested_user": get_tv_req['mediaInfo']['requests'][0]['requestedBy']['plexUsername'],
+                            "requested_date": get_tv_req['mediaInfo']['requests'][0]['requestedBy']['createdAt']
+                        },
+                        "time": now,
+                        "fields": {
+                            "hash": hash_id
+                        }
+                    }
+                )
+
+            if result['type'] == 'movie':
+                req = self.session.prepare_request(Request('GET', self.server.url + movie_endpoint + str(result['media']['tmdbId'])))
+                get_movie_req = connection_handler(self.session, req, self.server.verify_ssl)
+                hash_id = hashit(f'{get_movie_req["id"]}{get_movie_req["title"]}')
+
+                influx_payload.append(
+                    {
+                        "measurement": "Overseerr",
+                        "tags": {
+                            "type": "Requests",
+                            "server": self.server.id,
+                            "request_type": 1,
+                            "status": get_movie_req['mediaInfo']['status'],
+                            "title": get_movie_req['title'],
+                            "requested_user": get_movie_req['mediaInfo']['requests'][0]['requestedBy']['plexUsername'],
+                            "requested_date": get_movie_req['mediaInfo']['requests'][0]['requestedBy']['createdAt']
```

Comment:
```suggestion
                            "requested_date": get_movie_req['mediaInfo']['requests'][0]['media']['createdAt']
```
Actually it's grabbing "createdAt" of user not media

Reply:
These changes have been included in the latest commit on develop.